client/src/ui/organisms/placement/index.tsx

```diff
@@ -67,7 +78,14 @@ export function Placement() {
         throw Error(response.statusText);
       }
 
-      return (await response.json()) as PlacementStatus;
+      try {
+        const placementResponse: PlacementStatus | PlacementError =
+          await response.json();
+        gleanClick(`pong: pong->status ${placementResponse.status}`);
+        return placementResponse;
+      } catch (e) {
+        throw Error(response.statusText);
+      }
```

Comment:
Why do we catch and rethrow this error?

Reply:
we rename it, and keep how it was before